Model/Classes/User.cs

```diff
@@ -0,0 +1,51 @@
+﻿using AirportTicketBooking.Utils;
+
+namespace AirportTicketBooking.Model.Classes;
+public class User
+{
+    private static long _id = 0;
+    private List<Ticket> tickets = [];
+
+    public long Id { get; init; } = ++_id;
+    public required string Name { get; set; }
+    public required string Email { get; set; }
+    public required string Password { get; set; }
+
+    public bool BookTicket(Ticket ticket)
+    {       
+        tickets.Add(ticket);
+        return true;
+    }
+    public Ticket? RemoveTicket(Ticket ticket)
+    {
+        return tickets.Remove(ticket) ? ticket : null;
+    }
+    public List<Ticket> Tickets()
+    { return tickets; }
+    public UserRole Role { get; init; }
+
+    public User()
```

Comment:
I think this constructor is not needed because you add the required for all properties